lib/ReactViews/FeatureInfo/FeatureInfoDownload.jsx

```diff
@@ -14,7 +14,7 @@ import Styles from './feature-info-download.scss';
 const FeatureInfoDownload = createReactClass({
     propTypes: {
         data: PropTypes.object.isRequired,
-        name: PropTypes.string.isRequired,
+        baseFilename: PropTypes.string.isRequired,
```

Comment:
Technically, this is a breaking change.  If someone was previously using `FeatureInfoDownload` from their own code, it will no longer work after this change.  Admittedly this is a bit unlikely, so I'm not going to get too hung up on it.  But if the old name (`name`) doesn't seem too misleading, keeping it will avoid the problem.  Or, if you'd prefer to go with `baseFilename`, we need to mention the breaking change in CHANGES.md.